bin/physlr-make

```diff
@@ -1145,8 +1148,23 @@ g=$(shell awk '{x += $$2} END{print x}' $(name)/$(ref).fa.fai)
 	gunzip -c $< | awk '{ keep = $$6 " " $$8 " " $$9 != x; x = $$6 " " $$8 " " $$9 } keep' | $(gzip) >$@
 
 # Produce sequences in FASTA format from paths.
+ifeq ($(arcs), "false")
 %.$(draft).n10.sort.best.bed.path.fa: $(draft).fa %.$(draft).n10.sort.best.bed.path
 	$(time) $(python) $(bin)/physlr path-to-fasta --min-length=0 -V$V $^ >$@
+else
+%.$(draft).n10.sort.best.bed.path.fa: %.$(draft).n10.sort.best.bed.path $(draft).fa  $(draft)_c5_m50-6000_k30_r0.05_e30000_z500_pair.tsv
+	$(time) $(python) $(bin)/physlr path-to-fasta --min-length=0 \
+	--arcs-pair $(draft)_c5_m50-6000_k30_r0.05_e30000_z500_pair.tsv \
+	--dist-est $(draft)_c5_m50-6000_k30_r0.05_e30000_z500_dist.tsv \
+	-V$V $(draft).fa $< >$@
+endif
+
+$(draft)_c5_m50-6000_k30_r0.05_e30000_z500_pair.tsv: $(draft).fa $(lr).fq.gz
```

Comment:
I would say yes. Are we going to overload the help page with even more variables? I suppose I can just copy and paste the makefile help from arcs.

Reply:
Have you tried sweeping on those parameters at all? For example, `k`?

I'm fine with just deciding on parameters, it's just such a long prefix to have for the file if we aren't even going to be varying them.

Reply:
I dont feel like I should sweep. K for one dataset may not work on another. Im assuming 30 is by default because it worked well for the publication so I'm probably just gonna keep that.

Reply:
We actually used k=40-100 for the publication.

I don't mean sweep in every case, but in my opinion it is worth doing a few tests to see if there is an impact.

Reply:
I will change k in a separate PR when I finished benchmarking with various ks

Reply:
Ok sounds good